docs/docs/examples/ingestion/document_management_pipeline.ipynb

```diff
@@ -17,7 +17,7 @@
     "\n",
     "If we do not attach a vector store, we can only check for and remove duplicate inputs.\n",
     "\n",
-    "If a vector store is attached, we can also handle upserts! We have [another guide](/examples/ingestion/redis_ingestion_pipeline.ipynb) for upserts and vector stores."
+    "If a vector store is attached, we can also handle upserts! We have [another guide](/examples/ingestion/redis_ingestion_pipeline) for upserts and vector stores."
```

Comment:
I think this is still broken with this fix. Instead, I believe it ought to be:

`/en/stable/examples/ingestion/redist_ingestion_pipeline`

Reply:
I took care of this for us.